tests/setup_module_utils.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -euo pipefail
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+if [ ! -d "${1:-}" ] ; then
```

Comment:
This is already checked in `utils.sh`, therefore this log message will not appear if the script is already skipped because of the check in `utils.sh`. This does not seem to be intentional.

Reply:
I wanted to allow for this script being called separately in the future, in which case I wanted to leave the argument checking in.  It will also catch the case where someone makes an inadvertent breaking change to `tox.ini` where the environment variables are set.  So, I would prefer to leave this as-is.

Reply:
I do not follow how this will catch anything if `tox.ini` is wrong because utils.sh contains this check:

```bash
  if [ -n "$srcdir" -a -d "$srcdir" -a -n "$destdir" -a -d "$destdir" ]; then
￼    bash $TOPDIR/tests/setup_module_utils.sh "$srcdir" "$destdir"
￼  fi
```

So of `$srcdir` is not a directory, this script will not be called and there will also be no error message. Nevertheless, I guess in this case the CI should fail anyhow since this is just a helper to get the CI running and without it it should fail for the roles that enable it.